apps/happy-blocks/block-library/universal-header/header.scss

```diff
@@ -169,51 +169,51 @@ body.archive {
 		}
 	}
 
-	&.happy-blocks-header__learn {
-		margin-top: 120px;
-		margin-bottom: 70px;
+	&.happy-blocks-header {
+		&.is-learn {
+			margin-top: 120px;
+			margin-bottom: 70px;
 
-		@media ( max-width: $breakpoint-mobile ) {
-			margin: 56px 0;
-		}
-
-		.happy-blocks-global-header-site__title {
-			display: flex;
-			flex-direction: column;
-
-			.happy-blocks-global-header-site__title__wrapper {
-				margin-bottom: 32px;
-			}
-
-			h1 {
-				font-size: rem(70px);
-				line-height: 76px;
-				@media ( max-width: $breakpoint-mobile ) {
-					font-size: rem(50px);
-					line-height: 57px;
-				}
+			@media ( max-width: $breakpoint-mobile ) {
+				margin: 56px 0;
 			}
-			// subtitle
-			p {
-				margin-top: 16px;
-				font-size: rem(18px);
-				line-height: 26px;
-				@media ( max-width: $breakpoint-mobile ) {
-					font-size: 1rem;
-					line-height: 24px;
-				}
 
-			}
-			// search bar
-			form {
+			.happy-blocks-global-header-site__title {
 				display: flex;
-				justify-content: start;
+				flex-direction: column;
 
-				.happy-blocks-search__inside-wrapper {
-					input {
-						background-color: var(--studio-gray-0);
+				.happy-blocks-global-header-site__title__wrapper {
+					margin-bottom: 32px;
+				}
+
+				h1 {
+					font-size: rem(70px);
+					line-height: 76px;
+					@media ( max-width: $breakpoint-mobile ) {
+						font-size: rem(50px);
+						line-height: 57px;
 					}
 				}
+				// subtitle
+				p {
+					margin-top: 16px;
+					font-size: rem(18px);
+					line-height: 26px;
+					@media ( max-width: $breakpoint-mobile ) {
+						font-size: 1rem;
+						line-height: 24px;
+					}
+				}
+				// search bar
+				form {
+					display: flex;
+					justify-content: start;
+				}
+			}
+		}
+		.happy-blocks-search__inside-wrapper {
```

Comment:
Intentionally applying it to /support as well, it will look like this:
<img width="541" alt="image" src="https://github.com/Automattic/wp-calypso/assets/2653810/5f7bf55a-1d10-4417-a8df-b05d49412d29">